tests/unittests/aggregation_tests.py

```
@@ -668,15 +714,78 @@ def test_values(self):
             ],
         )
 
-        self.assertListEqual(
+        actual = d.values(
+            (F("predictions.detections[].label") != None).if_else(
```

Ruff (0.8.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)